src/Forms/GridField/GridFieldDetailForm.php

```diff
@@ -79,10 +88,14 @@ public function getURLHandlers($gridField)
      * controller who wants to display the getCMSFields
      *
      * @param string $name The name of the edit form to place into the pop-up form
+     * @param bool $showPagination Whether the `Previous` and `Next` buttons should display or not, leave as null to use default
+     * @param bool $showAdd Whether the `Add` button should display or not, leave as null to use default
      */
-    public function __construct($name = 'DetailForm')
+    public function __construct($name = null, $showPagination = null, $showAdd = null)
```

Comment:
Changing null to DetailForm would be a semver break, no?

Reply:
This should still yield the same results with the below line:

```
$this->setName($name ?: 'DetailForm');
```